sparselsh/lsh.py

```diff
@@ -58,6 +61,13 @@ class LSH(object):
     def __init__(self, hash_size, input_dim, num_hashtables=1,
                  storage_config=None, matrices_filename=None, overwrite=False):
 
+        assert isinstance(hash_size, int) and hash_size > 0, \
+            "hash_size must be a positive integer"
+        assert isinstance(input_dim, int) and input_dim > 0, \
+            "input_dim must be a positive integer"
+        assert isinstance(num_hashtables, int) and num_hashtables > 0, \
+            "num_hashtables must be a positive integer"
+
```

Comment:
Remove checks for int type. You can leave value checks.

Reply:
Fixed.